spec/std/file_spec.cr

```diff
@@ -82,7 +94,8 @@ describe "File" do
     idx.should eq(20)
   end
 
-  it "reads lines from file with each as iterator" do
+  # TODO: following two specs are broken on win32 because they leak file descriptors which break later tests
```

Comment:
No, ensure works fine you just can't assume that the GC runs between these specs and others. Relying on `File#ensure` is a hack. Regardless, the purpose of #6301 was to remove these methods because they always leak file references. And that's merged so this is all moot since these specs are gone.